suite-native/app/src/initActions.ts

```diff
@@ -19,6 +21,8 @@ export const applicationInit = createThunk(
         try {
             dispatch(initAnalyticsThunk());
 
+            dispatch(initMessageSystemThunk({ jwsPublicKey: getJWSPublicKey() }));
```

Comment:
It can be disabled for now as there is no implementation of banners, etc. So it is useless to fetch it right or at least the interval can be higher.

Reply:
very good point

Reply:
@Nodonisko @juriczech what do you prefer? Comment it out with explanation or temporary condition like `isDevelopOrDebugEnv()`? Or new static "feature flag" in .env files? Or just remove it completely for now and add it back only in the PR with the UI? I'll do whatever you want 🙂

Reply:
Just comment it out, we will uncomment it once we will have some UI for it.

Reply:
7cc0791fdc73cf24b9514f682690f581ed055b1a